pool/app/query/query.mli

```diff
@@ -0,0 +1,93 @@
+module Pagination : sig
+  module Limit : sig
```

Comment:
Please update the ones with the `include` only statement.

e.g.
`module Limit : Pool_common.Model.IntegerSig`